account-kit/react/src/components/auth/context.ts

```diff
@@ -3,6 +3,18 @@
 import type { Connector } from "@wagmi/core";
 import { createContext, useContext } from "react";
 
+export type AuthStepType =
```

Comment:
do we use this anymore? or in a dependent PR? If not then I'd say let's remove it, and pull it in when we need it.

Reply:
Removed, ty